interop/run.py

```diff
@@ -216,17 +217,29 @@ def run(self):
 if get_args().debug:
   console.setLevel(logging.DEBUG)
 else:
-  console.setLevel(logging.WARNING)
+  console.setLevel(logging.INFO)
 logger.addHandler(console)
 
+implementations = copy.deepcopy(IMPLEMENTATIONS)
+replace_arg = get_args().replace
+if replace_arg:
+  for s in replace_arg.split(","):
```

Comment:
This is annoying ... there should be a way to do multiple -r on the command line

Reply:
I'm not sure if this is possible with Python (or rather the Python library we're using).
If I specify multiple `-r` parameters, it looks like it's ignoring all but the last one.

Reply:
I just saw that this is possible using the `append` parameter: https://docs.python.org/3.7/library/argparse.html.
We can also use this for `--server` and `--client`. I'll prepare a separate PR.

Reply:
See #43.